src/scripts/router.js

```diff
@@ -142,7 +142,7 @@ export function initRouter(app) {
         ) {
             // Replace current page with target page in the browser history
             // so that we don’t break the back button
-            window.history.replaceState({}, '', `#${target}`)
+            window.history.replaceState({}, '', `/#${target}`)
```

Comment:
I'm afraid it will break the updater logic: https://github.com/Delegation-numerique-en-sante/mesconseilscovid/blob/master/src/scripts/updater.js#L132

What about removing the `source` parameter only if present?

Reply:
I think we don’t do redirects here so I don’t anticipate any problems, but yes we could be more specific and only remove source/ref/utm_xxx query parameters.